blackbox/docs/best_practice/full_restart_upgrade.txt

```diff
@@ -0,0 +1,175 @@
+.. highlight:: sh
+
+.. _full_restart_upgrade:
+
+====================
+Full Restart Upgrade
+====================
+
+The preferred way to upgrade a running CrateDB cluster is via the
+:ref:`rolling_upgrade` method. But this is not always an option (for example,
+when you are upgrading from one minor version to the next).
+
+When a :ref:`rolling_upgrade` is not possible, a full restart upgrade must be
+done if you want to update CrateDB.
+
+.. NOTE::
+
+   A full restart upgrade cannot be done without downtime for your database.
+
+.. CAUTION::
+
+   If you are using Docker, you must not run ``docker service update`` as this
+   performs a :ref:`rolling_upgrade`.
+
+   Instead you must remove the service and then recreate it.
+
+Upgrade Process
+===============
+
+.. WARNING::
+
+    Before upgrading, you should `back up your data`_.
+
+Stop Every Node
+---------------
+
+The first step is to shut down every CrateDB node in your cluster. By this, we
+mean, shut down every instance of the CrateDB daemon. The host operating system
+can be left running.
+
+Manually
+++++++++
+
+To stop the CrateDB daemon manually, run::
```

Comment:
hm, it seems that everything here is distribution dependent, why not just say you need to use the stop/start .... thingies as known in your actual setup?